TIL/daily/2025_03_20.md

```
@@ -0,0 +1,193 @@
+
+📍 **설명**
+
+- Flyable 인터페이스를 타입으로 선언했기 때문에 fly() 는 호출 가능하지만, Bird 의 다른 메서드들은 호출 불가능
+- 인터페이스를 타입으로 지정하면 해당 인터페이스의 기능만 사용 가능하므로, 기능에 대한 제약을 둘 수 있음.
+- 컴파일 타임 시점에서는 앞에 선언한 타입, 런타임 시점에서는 인스턴스의 타입으로 사용됨.
+
+⚠️ a***s*** **연산자** 
+
+```dart
+void main() {
+	Character character = Wizard('name', 10);
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)